tests/ceph_ansible/test_ansible_roll_over.py

```diff
@@ -145,6 +145,13 @@ def run(ceph_cluster, **kw):
     )
     out, rc = ceph_installer.exec_command(cmd=cmd, long_running=True)
 
+    if "4.2" in build:
```

Comment:
can we have this statement,
```
if build == "4.2":
```

Reply:
build will have 4.2-rhel-7 or 4.2-rhel-8 so this condition will fail

Reply:
We could have `if build.startswith("4.2")` not that the current implementation wouldn't work. I prefer not to change it to `build == '4.2'` now.

Reply:
> build will have 4.2-rhel-7 or 4.2-rhel-8 so this condition will fail

Sorry, Just had pipeline v2.0 in mind. 
Make sense to hold back to current logic.